pkg/operators/v1alpha1/subscription_types.go

```diff
@@ -111,6 +111,12 @@ const (
 
 	// SubscriptionResolutionFailed indicates that the dependency resolution in the namespace in which the subscription is created has failed
 	SubscriptionResolutionFailed SubscriptionConditionType = "ResolutionFailed"
+
+	// SubscriptionBundleUnpacking indicates that the unpack job is currently running
+	SubscriptionBundleUnpacking SubscriptionConditionType = "BundleUnpacking"
+
+	// SubscriptionBundleUnpackFailed indicates that the unpack job failed
+	SubscriptionBundleUnpackFailed SubscriptionConditionType = "BundleUnpackFailed"
```

Comment:
Are these used anywhere?

Reply:
@tmshort will be used in https://github.com/operator-framework/operator-lifecycle-manager/pull/2942